src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpSyntaxFormattingService.cs

```diff
@@ -63,4 +50,16 @@ protected override IFormattingResult CreateAggregatedFormattingResult(SyntaxNode
         protected override AbstractFormattingResult Format(SyntaxNode node, SyntaxFormattingOptions options, IEnumerable<AbstractFormattingRule> formattingRules, SyntaxToken startToken, SyntaxToken endToken, CancellationToken cancellationToken)
             => new CSharpFormatEngine(node, options, formattingRules, startToken, endToken).Format(cancellationToken);
     }
+
+#if !CODE_STYLE
+    [ExportLanguageService(typeof(ISyntaxFormattingService), LanguageNames.CSharp), Shared]
+    internal class CSharpSyntaxFormattingService : CSharpSyntaxFormatting, ISyntaxFormattingService
```

Comment:
Can we just move this class directly to Workspaces layer and avoid the `#if !CODE_STYLE`?

Reply:
yes.  that makes sense.  will do in followup!